Solutions/ESI - Exchange Security Config and Audit Insights/SolutionMetadata.json

```diff
@@ -13,5 +13,8 @@
 	  "email": "support@microsoft.com",
 	  "tier": "Microsoft",
 	  "link": "https://support.microsoft.com"
+	},
+	"author": {
+		"name": "Microsoft"
 	}
```

Comment:
Hi @nlepagnez,
Can you remove the author details from SolutionMetadata file.
Thanks